frontend/app/model/Nav.scala

```diff
@@ -10,6 +10,8 @@ object Nav {
     subNav: Seq[NavItem] = Nil
   )
 
+  case class NavAnchor(href: String, title: String)
```

Comment:
this case class is only used in the `views` package, and should be contained within it.